src/richie/apps/courses/admin.py

```diff
@@ -2,17 +2,40 @@
 Courses application admin
 """
 from django.contrib import admin
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
 
 from cms.extensions import PageExtensionAdmin
 
+from .fields import CourseRunSplitDateTimeField
 from .forms import LicenceFormAdmin
-from .models import Course, Licence, Organization
+from .models import Course, CourseRun, Licence, Organization
+from .widgets import CourseRunSplitDateTimeWidget
+
+
+class CourseRunInline(admin.StackedInline):
```

Comment:
You may try `admin.TabularInline` it is commonly more clean and accessible

Reply:
Yes that's what I used at first but it requires some adaptation to fit in a line so as a first approach I switched to stacked inlines...

![image](https://user-images.githubusercontent.com/1427165/44852442-8b15c680-ac63-11e8-8961-70fe4c026fc6.png)

Reply:
Ah yeah, table makes troubleshooting with non text element like the date widtget.